resume-builder/src/TestComponent.js

```diff
@@ -95,30 +95,31 @@ class TestComponent extends Component {
     }]
   }
   // TODO
-  // 1. Use immutability-helper to handle nested data
+  // 1. Use immutability-helper package to handle nested data
```

Comment:
Helpful list for reviewers to follow your line of thought/understand how you tackled the prompt!